src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -571,7 +571,7 @@ private static Float DotProduct(Float[] a, int aOffset, Float[] b, int[] indices
 
         private static class Mkl
         {
-            private const string DllName = "Microsoft.ML.MklImports.dll";
+            private const string DllName = "Microsoft.MachineLearning.MklImports.dll";
```

Comment:
>Microsoft.MachineLearning [](start = 44, length = 25)

Is this related to the issue raised?

Given that we decided on `Microsoft.ML` vs. the more verbose `Microsoft.MachineLearning`, should not that DLL change its name, rather than the other way around?

Reply:
I had the same question but I also feel this should be made as a separate change and we should open an issue for this.

---
In reply to: [187159434](https://github.com/dotnet/machinelearning/pull/102#discussion_r187159434) [](ancestors = 187159434)